tests/layer_tests/pytorch_tests/test_pythonop.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class TestPythonOp(PytorchLayerTest):
+    def _prepare_input(self):
+        return (np.random.randn(1, 3, 128, 128),)
+
+    def create_model(self):
+        import torch
+        from torch.autograd.function import Function
+
+        class _ExpF(Function):
+            @staticmethod
+            def forward(ctx, input_tensor):
+                exp = torch.exp(input_tensor)
+                ctx.save_for_backward(exp)
+                return exp
+
+            @staticmethod
+            def backward(ctx, output_grad):
+                (result,) = ctx.saved_tensors
+                return output_grad * result
+
+        exp_f = _ExpF.apply
+
+        class prim_pythonop(torch.nn.Module):
+            def forward(self, input_tensor):
+                return torch.add(exp_f(input_tensor), 1)
```

Comment:
Could you also add test for DeformableConvolution that we have in models?

Reply:
This op would add `torchvision::deform_conv2d` into unconverted OP, adding test would require to enable this op as well. I think it might be better to enable it in different PR to not mix it up, but if you want, I could enable this in current PR